pyracf/common/alter_operation_error.py

```diff
@@ -0,0 +1,24 @@
+"""Exception to use when Alter operation would add a new profile."""
+
+
+class AlterOperationError(Exception):
+    """
+    Raised when a profile passed into an Alter is not successfully extracted.
+    """
+
+    def __init__(self, profile_name: str, class_name: str) -> None:
```

Comment:
Make same updates to here that I suggested for `AddOperationError`.

Reply:
Agree, that's cleaner. Change made across the board. Corresponding changes also made in unit testing where the error contents are evaluated.